api/database/helpers/EnumsForFactories.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Database\Helpers;
+
+// TODO: any way to pull these directly from the graphql schema?
```

Comment:
I don't love this but I like it better than pasting string literals in all the factories and seeders.  Any way to pull it from the schema?  I can create an issue to investigate.